src/util/types.ts

```diff
@@ -24,6 +24,10 @@ export interface Contracts {
 	vesting: MerkleTokenVesting;
 }
 
+export interface GrantContract {
+	vesting: TokenVestingController;
```

Comment:
Should be inside of the `Contracts` interface. You could call them `grantableVesting`